data/campaigns/Eastern_Invasion/utils/items.cfg

```diff
@@ -904,6 +904,20 @@ plague_staff #enddef
             apply_to=overlay
             add=misc/plague-staff-icon.png
         [/effect]
+        
```

Comment:
Looks like the check is complaining about trailing white-space here, but GitHub view isn't rendering any.

Reply:
Yeah, I can only see it if I look at the raw text, not what GitHub renders.